Styling Subscribe Block

rccsousa opened the pull request:
Why:
- The **Subscribe to Subvisual Inspo** Call to Action block wasn't styled;
How:
- Added styling elements to the block;
- Added images to repository for future use in different sections;

rccsousa committed:
style subscribe card block

rccsousa committed:
add images

rccsousa committed:
Merge branch 'main' of https://github.com/subvisual/content-sub into rs/format-subscribe-block

rccsousa committed:
remove files that were out of scope for this PR

rccsousa committed:
Merge branch 'main' of https://github.com/subvisual/content-sub into rs/format-subscribe-block

rccsousa committed:
fix image

rccsousa committed:
update button

rccsousa committed:
update margin-top

rccsousa committed:
Merge branch 'main' into rs/format-subscribe-block

davelange commented on src/app/_blocks/Subscribe/index.tsx:
```diff
@@ -1,11 +1,19 @@
-        <button>Subscribe</button>
+    <div className={styles.container}>
+      <div className={styles.imageContainer}>
+        <img className={styles.image} src={'/subscribe-card-image.png'} />
```
Something to look into eventually: https://nextjs.org/docs/app/api-reference/components/image
It's a component that handles things like optimization, etc.

davelange commented:
![Screenshot 2024-09-27 at 11 54 06](https://github.com/user-attachments/assets/e274a2b5-e63b-4af0-92b3-9e0ad030a148)

The fonts aren't quite there yet. Looking at fonts.css again, you have Colfax, but no Colfax medium (sorry, I don't think I explained this clearly). What you need is:

```
@font-face {
  font-family: 'Colfax';
  src: url('/fonts/Colfax.woff2') format('woff2'),
  url('/fonts/Colfax.woff') format('woff');
  font-weight: normal;
  etc
}

@font-face {
  font-family: 'Colfax';
  src: url('/fonts/ColfaxMedium.woff2') format('woff2') etc
  font-weight: bold;
  etc
}
```

Then you can use it as:
```
body {
  font-family: var(--colfax)
  font-weight: normal <-- will use the regular font by default
}

...

h5 {
 font-weight: bold <-- will use colfax bold
}
```

rccsousa committed:
format and fix

- updated font variables to reflect changes to globals.css

rccsousa commented:
@davelange thanks for clarifying! i'll patch it on separate fix to globals.css
i've updated the font variables to reflect ( didn't update `--colfax-regular` to just `--colfax` )

i've also changed the margins to reflect the intended design.

i believe the end result is now correct:

desktop:
![image](https://github.com/user-attachments/assets/d38c36c7-29ad-48ed-a476-bd3698b14fcc)

mobile:
![image](https://github.com/user-attachments/assets/1b5d02d8-1b07-4ad9-b35b-8f18aa9ba360)
